solutions/Hawk/solution.ini

```diff
@@ -1,6 +1,6 @@
 [build]
-default=mvn -B --quiet -f org.hawk.ttc2018/pom.xml package
-skipTests=mvn -B --quiet -f org.hawk.ttc2018/pom.xml package -DskipTests
+default=./build-hawk-neo4j.sh && mvn -B --quiet -f org.hawk.ttc2018/pom.xml package && mvn -B --quiet -f org.hawk.ttc2018/pom.xml test -Dbackend=neo4j
+skipTests=./build-hawk-neo4j.sh && mvn -B --quiet -f org.hawk.ttc2018/pom.xml package -DskipTests
```

Comment:
Do we need a `skipTests` variant also for Neo4j?

Reply:
No, because there's no Neo4j-specific code to compile. Essentially, the solution uses reflection to construct either a `Neo4JDatabase` or a `LevelDBGreycatDatabase`, so it's only a matter of swapping in the right libraries and specifying which class to instantiate in an environment variable. It still falls back on `LevelDBGreycatDatabase` if the env var is not set.